fetch.bs

```diff
@@ -813,23 +802,28 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn for=export>script-like destination</dfn> is "<code>script</code>",
+"<code>serviceworker</code>", "<code>sharedworker</code>", or "<code>worker</code>".
```

Comment:
Would this be more clear as `A [=request/destination=] is <dfn for="request/destination" export lt="script-like destination">script-like</dfn> if it is one of ..., ..., or ...`?